libseccomp/src/filter_context.rs

```diff
@@ -201,10 +200,9 @@ impl ScmpFilterContext {
     /// ctx.remove_arch(ScmpArch::X86)?;
     /// # Ok::<(), Box<dyn std::error::Error>>(())
     /// ```
-    pub fn remove_arch(&mut self, arch: ScmpArch) -> Result<bool> {
+    pub fn remove_arch(&mut self, arch: ScmpArch) -> Result<()> {
         match unsafe { seccomp_arch_remove(self.ctx.as_ptr(), arch.to_sys()) } {
-            0 => Ok(true),
-            MINUS_EEXIST => Ok(false),
```

Comment:
Could you add the following comment?

```rust
// Similar to add_arch, -EEXIST is returned if the arch is not present.
// Succeed silently in that case, this is not fatal and the architecture
// is not present in the filter after remove_arch
```

Besides, please remove the following doc comments.
https://github.com/libseccomp-rs/libseccomp-rs/pull/218/files#diff-cf472dd53e08212cd16fb7c9fa788b407cae006cb94ed5b7249850c6026618edR178-R180